internal/manifest/parser/template_to_module.go

```diff
@@ -67,27 +67,27 @@ func (p *Parser) GenerateMandatoryModulesFromTemplates(ctx context.Context,
 			moduleName = template.Name
 		}
 
-		modules = p.appendModuleWithInformation(templatelookup.AvailableModule{
+		modules = p.appendModuleWithInformation(templatelookup.ModuleStatusInfo{
 			Module: v1beta2.Module{
 				Name:                 moduleName,
 				CustomResourcePolicy: v1beta2.CustomResourcePolicyCreateAndDelete,
 			},
-			Enabled: true,
+			IsEnabled: true,
 		}, kyma, template, modules)
 	}
 
 	return modules
 }
 
-func (p *Parser) appendModuleWithInformation(module templatelookup.AvailableModule, kyma *v1beta2.Kyma,
+func (p *Parser) appendModuleWithInformation(module templatelookup.ModuleStatusInfo, kyma *v1beta2.Kyma,
 	template *templatelookup.ModuleTemplateInfo, modules common.Modules,
 ) common.Modules {
 	if template.Err != nil && !errors.Is(template.Err, templatelookup.ErrTemplateNotAllowed) {
 		modules = append(modules, &common.Module{
 			ModuleName:  module.Name,
 			Template:    template,
-			Enabled:     module.Enabled,
-			IsUnmanaged: module.Unmanaged,
+			Enabled:     module.IsEnabled,
+			IsUnmanaged: module.IsUnmanaged,
```

Comment:
Thanks for starting to align these! Personally, I would vote for the other way round though (just `Enabled` and `Unmanaged`). We typically use `IsSomething` for methods, and just `Something` for the attribute.

See e.g., here https://github.com/kyma-project/lifecycle-manager/blob/31dca4c4202dc903b04199c232d397bf9abd6ddc/api/v1beta2/modulereleasemeta_types.go#L71-L77 or here https://github.com/kyma-project/lifecycle-manager/blob/31dca4c4202dc903b04199c232d397bf9abd6ddc/api/v1beta2/kyma_types.go#L391-L399

I know that this one `IsUnmanaged` occurrence is coming out of common.Module. If it is not a big deal we could include change htis here. If so, let's rather do a separate PR.


func (kyma *Kyma) IsBeta() bool {
	beta, found := kyma.Labels[shared.BetaLabel]
	return found && shared.IsEnabled(beta)
}